src/progress-bar.tsx

```diff
@@ -35,11 +35,11 @@ export const ProgressBar = ({
     const isInverse = useIsInverseVariant();
     const progressValue = Math.max(0, Math.min(100, progressPercent));
 
-    const getFormattedLabel = () => {
-        const ariaLabelText = ariaLabel || texts.loading || t(tokens.loading);
-        const completedLabelText = texts.progressBarCompletedLabel || t(tokens.progressBarCompletedLabel);
+    const getFormattedValueText = () => {
+        const name = ariaLabel || texts.loading || t(tokens.loading);
+        const completedText = texts.progressBarCompletedLabel || t(tokens.progressBarCompletedLabel);
 
-        return `${ariaLabelText}, ${progressValue}% ${completedLabelText}`;
+        return `${name}, ${progressValue.toFixed(1)}% ${completedText}`;
```

Comment:
I've rounded the value to 1 decimal to avoid screen reader reading "16.66666666666" for 1/6, for example.